.bazelci/tests.yml

```diff
@@ -74,3 +74,6 @@ tasks:
     <<: *rolling
     build_flags:
       - "--enable_bzlmod"
+    build_targets:
+      - "//distro:distro"
+      - "//distro:relnotes"
```

Comment:
Looks good. Were this distros being produced manually before or am I missing something?

Reply:
They were not being produced with --enable_bzlmod before. Now they are tested both with and without bzlmod.